src/GafferArnold/ArnoldOptions.cpp

```diff
@@ -86,6 +86,9 @@ ArnoldOptions::ArnoldOptions( const std::string &name )
 	// Subdivision
 
 	options->addChild( new Gaffer::NameValuePlug( "ai:max_subdivisions", new IECore::IntData(999), false, "maxSubdivisions" ) );
+	options->addChild( new Gaffer::NameValuePlug( "ai:subdiv_dicing_camera", new IECore::StringData( "" ), false, "subdivisionDicingCamera" ) );
+	options->addChild( new Gaffer::NameValuePlug( "ai:subdiv_frustum_culling", new IECore::BoolData( false ), false, "subdivisionFrustumCulling" ) );
+	options->addChild( new Gaffer::NameValuePlug( "ai:subdiv_frustum_padding", new IECore::FloatData( 0.0f ), false, "subdivisionFrustumPadding" ) );
```

Comment:
Looking at ArnoldAttributes and also the rest of this file, our convention for deriving a plug name from an Arnold name seems to be to do a simple translation from snake_case to camelCase, so the plug names should be `subdivDicingCamera` rather than `subdivisionDicingCamera` and so on.